ruleset.xml

```diff
@@ -386,7 +387,7 @@
       <property name="minimum" value="3" />
     </properties>
   </rule>
-  <rule ref="rulesets/java/naming.xml/MisleadingVariableName">
+  <rule ref="rulesets/java/naming.xml/MIsLeadingVariableName">
```

Comment:
Is this a typo or is this on purpose?

Reply:
Yeah, a typo, but on PMD's side I'm afraid :cry: